test/acceptance/test/add_tests.go

```diff
@@ -870,6 +875,58 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 			Eventually(listOutput).Should(ContainSubstring(appName1))
 			Eventually(listOutput).Should(ContainSubstring(appName2))
 		})
+
+		By("When I pause an app: "+appName1, func() {
+			pauseOutput, _ = runCommandAndReturnStringOutput(WEGO_BIN_PATH + " app pause " + appName1)
+		})
+
+		By("Then I should see pause message", func() {
+			Expect(pauseOutput).To(ContainSubstring("gitops automation paused for " + appName1))
+		})
+
+		By("And changes to the app files should not be synchronized", func() {
+			appManifestFile1, _ = runCommandAndReturnStringOutput("cd " + repoAbsolutePath1 + " && ls")
```

Comment:
small nit: it might be a good idea to assert the error here. If this fails for some reason we probably won't know why it failed.